src/linker/Linker/UnconditionalSuppressMessageAttributeState.cs

```diff
@@ -22,7 +22,7 @@ public UnconditionalSuppressMessageAttributeState (LinkContext context)
 			InitializedAssemblies = new HashSet<AssemblyDefinition> ();
 		}
 
-		public void AddSuppression (CustomAttribute ca, ICustomAttributeProvider provider)
```

Comment:
Nit: Now that we moved the CA walk here, should we make `TypeRefHasUnconditionalSuppressions` private as well (same applies to `LookForModuleLevelAndGlobalSuppressions`.